api/src/opentrons/protocols/advanced_control/transfers.py

```diff
@@ -571,6 +571,12 @@ def _plan_distribute(self):
                .. Dispense air gap -> ...*
 
         """
+
+        if self._strategy.disposal_volume >= self._instr.max_volume:
+            raise ValueError(
+                "The Disposal Volume must be less than the Maximum Volume of the Instrument"
```

Comment:
Looks good! We can probably lowercase all these terms in the error text:

```suggestion
                "The disposal volume must be less than the maximum volume of the pipette"
```